src/Makefile.am

```diff
@@ -62,6 +62,7 @@ PYTHON_BRIDGE_FILES = \
 	src/cockpit/channels/dbus.py \
 	src/cockpit/channels/dbus_internal.py \
 	src/cockpit/channels/filesystem.py \
+	src/cockpit/channels/http_stream2.py \
```

Comment:
My impulse would have been to call this `http.py`.  Compare that it's not `dbus_json3` or so.

Reply:
Renamed to http.py.